sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -2178,6 +2236,8 @@ private void ValidateFormPage(FormPage formPage, bool includeFieldElements, int
                 Assert.AreEqual(expectedPageNumber, table.PageNumber);
                 Assert.Greater(table.ColumnCount, 0);
                 Assert.Greater(table.RowCount, 0);
+                // Currently not true for CustomForms. Issue https://github.com/Azure/azure-sdk-for-net/issues/16827
```

Comment:
Just curious, for supervised or unsupervised is it not sent?

Reply:
From my tests, both come back without table bounding box